eclair-core/src/main/scala/fr/acinq/eclair/NodeParams.scala

```diff
@@ -610,7 +611,8 @@ object NodeParams extends Logging {
       revokedHtlcInfoCleanerConfig = RevokedHtlcInfoCleaner.Config(
         batchSize = config.getInt("db.revoked-htlc-info-cleaner.batch-size"),
         interval = FiniteDuration(config.getDuration("db.revoked-htlc-info-cleaner.interval").getSeconds, TimeUnit.SECONDS)
-      )
+      ),
+      wakeUpTimeout = 30 seconds,
```

Comment:
Because it's too early to introduce a wake-up config at that point, I'm not sure what else we'll want there. Remember that this is `eclair/master` so anything we introduce to the config needs to handle backwards-compatibility!